### PR TITLE
MGMT-15698: Document how to add custom manifests

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -222,6 +252,40 @@ Configure the disk to use `/path/to/appliance.raw`
...
+* Note that any manifest added here will apply to **only** to the cluster installed using this config-iso.
+* Find more details and additional examples in OpenShift documentation: [Customizing nodes](https://docs.openshift.com/container-platform/4.13/installing/install_config/installing-customizing.html).
+
+1. Create the openshift manifests directory
+```shell
+mkdir $APPLIANCE_ASSETS:/openshift
+```
+
+2. Add one or more custom manifests under `$CLUSTER_CONFIG:/openshift`. For example:
+```yaml
```
Comment (danielerez) [inline docs/user-guide.md:266]: Not sure we need another example here. The docs should cover it for the user flow.
Just add also this reference for 'Post-installation configuration': https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html#using-machineconfigs-to-change-machines

File: docs/user-guide.md [hunk 2]
```diff
@@ -135,6 +135,36 @@ sshKey: <redacted>
 userCorePass: <redacted>
 ```
 
+### Add custom manifests (Optional)
+* Note that any manifest added here will apply to **any** of the clusters installed using this image.
+* Find more details and additional examples in OpenShift documentation: [Customizing nodes](https://docs.openshift.com/container-platform/4.13/installing/install_config/installing-customizing.html).
+
```
Comment (danielerez) [inline docs/user-guide.md:141]: Also some example for 'Post-installation configuration': https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html#using-machineconfigs-to-change-machines

File: docs/user-guide.md (hunk 1 above)
Comment (nmagnezi) [reply docs/user-guide.md]: Done

File: docs/user-guide.md (hunk 2 above)
Comment (nmagnezi) [reply docs/user-guide.md:143]: Done

File: docs/user-guide.md (hunk 1 above)
Comment (nmagnezi) [reply docs/user-guide.md]: Added a cross reference to the above example instead.

**Pull request merged**